app/view/index.html

```diff
@@ -0,0 +1,149 @@
+    <title>Homework #5</title>
+    <link rel="stylesheet" href="../public/stylesheets/all.css">
+</head>
+<body>
+    <div class="page-header">
+        <nav class="page-header_nav_section">
+            <div class="page-header_nav_section_item page-header_nav_section-freelancer">
+                <a href="#">Freelancer</a>
+            </div>
+            <div class="page-header_nav-toggler">
+                <span>&#9776;</span>
+            </div>
+            <div class="page-header_nav_section_item m-auto">
+                <ul class="page-header_nav_list">
+                    <li class="page-header_nav_list_item">
+                        <a class="page-header_nav_list_item_link" href="#">about</a>
+                    </li>
+                    <li class="page-header_nav_list_item">
+                        <a class="page-header_nav_list_item_link" href="#">work</a>
+                    </li>
+                    <li class="page-header_nav_list_item">
+                        <a class="page-header_nav_list_item_link" href="#">portfolio</a>
+                    </li>
+                    <li class="page-header_nav_list_item">
+                        <a class="page-header_nav_list_item_link" href="#">contact</a>
+                    </li>
+                    <li class="page-header_nav_list_item">
+                        <a href="#" class="page-header_nav_list_item_link">
+                            <img src="../src/images/icons/Twitter.png" alt="twitter icon">
+                        </a>
+                    </li>
+                    <li class="page-header_nav_list_item">
+                        <a href="#" class="page-header_nav_list_item_link">
+                            <img src="../src/images/icons/Facebook.png" alt="facebook icon">
+                        </a>
+                    </li>
+                </ul>
+            </div>
+        </nav>
+        <header class="page-header_main">
+            <section class="page-header_main_centered-section">
+                <h1 class="page-header_main_large_text">
+                    intelligence <br/> is nothing without <br/> ambition<span class="red-dot"></span>
+                </h1>
+                <p class="page-header_main_description">We`re small but impressive creative agency.</p>
+                 <button class="page-header_main_rounded-button">read more</button>
+            </section>
+        </header>
+    </div>
+    <main class="main-content">
+        <section class="main-content_advantages">
+            <article class="main-content_advantages_article">
+                <h2 class="main-content_advantages_header">peace<br/>of mind<span class="red-dot"></span></h2>
+                <p class="main-content_advantages_description">You`re safe with us. The privacy and security of our financial information is our top privacy.</p>
+                <button class="main-content_advantages_more-button">more &#8594</button>
```

Comment:
Упустил точку с комой. Смотри похожие места. Не валидный html получается.
```suggestion
                <button class="main-content_advantages_more-button">more &#8594;</button>
```